mender_grubenv_defines.example

```diff
@@ -18,5 +18,11 @@ kernel_imagetype=bzImage
 # Optional
 ################################################################################
 
+# Partition index of kernel A
+mender_kernela_part=5
+
+# Partition index of kernel B
+mender_kernelb_part=6
```

Comment:
Since separate kernel partitions are not supported by stock Mender, I think these should be changed a bit:

* They should be commented out, since the common case is to not define them, and it would misbehave if they are defined.
* The comment should mention that you need special arrangement to use this, and should point to an Update Module which performs the update of the kernel partitions in addition to the rootfs. Which one are you using?

Reply:
Updated.

Currently integrating on hardware w/ encrypted rootfs and data partitions. Not ready yet, but I'll open-source the layer, Update Modules, etc when stable (hopefully soon).

Reply:
Ok, great! I'll hold this back for now, as I don't think we should publish only half of the solution. Let me know when you have published the Update Module, and we can add the comment here.

The rest looks good to me now!

Reply:
I've published a layer that contains the kernel partitioning and update logic (via state-script) here: https://github.com/coreycothrum/meta-mender-kernel